inc/php_basic_auth/namespace.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Altis\Security\PHP_Basic_Auth;
+
+use function Altis\get_config;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'hmauth_action_before_dev_env_check', __NAMESPACE__ . '\\check_environment_credentials' );
```

Comment:
Almost missed this too - should this hook be updated to `define_credentials` now instead?